tests/test_parasha.py

```diff
@@ -0,0 +1,12 @@
+from app.internal import load_parasha
+from sqlalchemy.orm import Session
+
+
+def test_if_db_correct(session: Session):
+    first_parasha = [0, 'Parashat Vayechi', 'פרשת ויחי',
+                     'https://www.hebcal.com/sedrot/vayechi-20210102?',
+                     'i=on&utm_source=js&utm_medium=api',
+                     '2021-01-02']
```

Comment:
Split rows after the first "[" and after each ",", including last.

Reply:
okk